chrome_extension/background.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @file background.js
+ * @description Creates a new background script that listens for messages from the content script
+ * and sends them to a WebSocket server.
+*/
+
+let socket;
+let TIMEOUT_INTERVAL = 5000; // ms
+
+/*
+ * Function to connect to the WebSocket server.
+*/
+function connectWebSocket() {
+  // TODO: Ideally we read "ws://localhost:8765" and `TIMEOUT_INTERVAL` from config.py, or it gets passed in somehow. 
+  socket = new WebSocket("ws://localhost:8765");
```

Comment:
Can you please define the port as a named constant next to TIMEOUT_INTERVAL, and move the comment there and update it accordingly?

Reply:
Done here: fad41434f9951fddd88b851a472a8c32c128ff56